src/components/Layout/Header/ActionCenter/components/Details/StreamingSwapDetails.tsx

```diff
@@ -14,48 +14,46 @@ type StreamingSwapDetailsProps = {
 }
 
 export const StreamingSwapDetails: React.FC<StreamingSwapDetailsProps> = ({ swap }) => {
-  const translate = useTranslate()
+  const streamingSwapData = useStreamingProgress({ swap })
 
-  const streamingProgress = useStreamingProgress({ swap })
+  const { totalSwapCount: streamingTotalSwapCount, successfulSwapCount } = streamingSwapData ?? {}
 
-  const {
-    numSuccessfulSwaps: thorchainNumSuccessfulSwaps,
-    totalSwapCount: thorchainTotalSwapCount,
-    isComplete: isThorchainStreamingComplete,
-  } = streamingProgress ?? {}
+  const translate = useTranslate()
 
-  const isComplete = isThorchainStreamingComplete || swap.status === SwapStatus.Success
+  const isSwapComplete = swap.status === SwapStatus.Success
 
-  const totalSwapCount = useMemo(() => {
-    return thorchainTotalSwapCount === 0 ? 1 : thorchainTotalSwapCount
-  }, [thorchainTotalSwapCount])
+  const progress = useMemo(() => {
+    if (isSwapComplete) return 100
+    if (!streamingTotalSwapCount) return 0
 
-  const numSuccessfulSwaps = useMemo(() => {
-    if (thorchainNumSuccessfulSwaps === 0 && isComplete) return 1
+    return bnOrZero(successfulSwapCount)
+      .div(bnOrZero(streamingTotalSwapCount))
+      .multipliedBy(100)
+      .toNumber()
+  }, [successfulSwapCount, streamingTotalSwapCount, isSwapComplete])
 
-    return thorchainNumSuccessfulSwaps
-  }, [isComplete, thorchainNumSuccessfulSwaps])
+  const maxSwapCount = useMemo(() => {
+    if (swap.status === SwapStatus.Success) return successfulSwapCount
 
-  const progress = useMemo(() => {
-    return isComplete
-      ? 100
-      : bnOrZero(numSuccessfulSwaps).div(bnOrZero(totalSwapCount)).multipliedBy(100).toNumber()
-  }, [numSuccessfulSwaps, totalSwapCount, isComplete])
+    return streamingTotalSwapCount ?? 1
+  }, [streamingTotalSwapCount, successfulSwapCount, swap.status])
 
   if (!swap.isStreaming) return null
 
   return (
-    <Row fontSize='sm'>
+    <Row fontSize='sm' alignItems='center'>
       <Row.Label>{translate('notificationCenter.streamingStatus')}</Row.Label>
       <Row.Value display='flex' alignItems='center' gap={2}>
         <Progress
           width='100px'
           size='xs'
           value={progress}
-          colorScheme={isComplete ? 'green' : 'blue'}
+          colorScheme={isSwapComplete ? 'green' : 'blue'}
+          isAnimated={true}
+          hasStripe={isSwapComplete ? false : true}
         />
         <RawText>
-          ({numSuccessfulSwaps}/{totalSwapCount})
+          ({successfulSwapCount}/{maxSwapCount})
```

Comment:
And then if we go with `finalSwapCount` (which it feels like we would like?), we could do `successfulSwapCount}/{finalSwapCount || maxSwapCount}

Reply:
I'm not 100% aligned on this, as the finalSwapAccount is just the success swap counts of a finished streaming swap... I feel like we probably don't want to do that, but if you're really attached to it, I would make the effort...

It wouldn't be so big to implement, inside every streaming swap hooks, if the swap is success, push a new metadata "finalSwapCount" with the success swap numbers, but considering the current usage of this, it's not a big deal right now, if we shot the final swap count in more places (we probably will never do this), then why not